packages/plans-grid-next/src/components/plan-type-selector/components/interval-type-toggle.tsx

```diff
@@ -60,21 +60,9 @@ export const IntervalTypeToggle: React.FunctionComponent< IntervalTypeProps > =
 		}
 	}
 
-	const additionalPathProps = {
-		...( props.redirectTo ? { redirect_to: props.redirectTo } : {} ),
-		...( props.selectedPlan ? { plan: props.selectedPlan } : {} ),
-		...( props.selectedFeature ? { feature: props.selectedFeature } : {} ),
-	};
-
-	const isDomainUpsellFlow = new URLSearchParams( window.location.search ).get( 'domain' );
-
-	const isDomainAndPlanPackageFlow = new URLSearchParams( window.location.search ).get(
-		'domainAndPlanPackage'
-	);
-
-	const isJetpackAppFlow = new URLSearchParams( window.location.search ).get( 'jetpackAppPlans' );
-
-	const intervalTabs = showBiennialToggle ? [ 'yearly', '2yearly' ] : [ 'monthly', 'yearly' ];
+	const intervalTabs: SupportedUrlFriendlyTermType[] = showBiennialToggle
```

Comment:
Checking in a quick follow-up

Reply:
Hey gonna try to summarize to see if I understand your comment properly.

I think you're suggesting that the `showBiennialToggle` condition is no longer necessary because, in that instance, we'd display the multi-year dropdown instead?

Reply:
Oh I think I see what you're saying. 

The type for `intervalTabs` is interpreted as a string[] if we don't explicitly set something, which contends with the type requirements downstream in the `onPlanIntervalUpdate` method.

Perhaps `SupportedUrlFriendlyTermType` is too broad, since it includes `3yearly` as a valid type, which is never included in `intervalTabs`. Either way, I see some type definition as necessary